src/reanimated2/Hooks.js

```diff
@@ -184,7 +184,7 @@ function styleDiff(oldStyle, newStyle) {
   return diff;
 }
 
-function styleUpdater(viewTag, updater, state, maybeViewRef) {
+function styleUpdater(viewTag, updater, state, maybeViewRef, viewName) {
```

Comment:
I still think it'd be more clear if we stopped using pair viewTag and viewName as separate arguments and just rename `viewTag` to `viewDescriptor` here and in other places. If not we should at least place them next to each other

Reply:
fixed b3f674c646c6d7972e8a0ed50226c8028433ac16